sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -430,21 +431,31 @@ private synchronized AnalyticsTransmissionTarget getInstanceTransmissionTarget(S
                 AppCenterLog.debug(LOG_TAG, "Returning transmission target found with token " + transmissionTargetToken);
                 return transmissionTarget;
             }
-            transmissionTarget = new AnalyticsTransmissionTarget(transmissionTargetToken, null);
-            AppCenterLog.debug(LOG_TAG, "Created transmission target with token " + transmissionTargetToken);
+            transmissionTarget = createAnalyticsTransmissionTarget(transmissionTargetToken);
             mTransmissionTargets.put(transmissionTargetToken, transmissionTarget);
-            final AnalyticsTransmissionTarget finalTransmissionTarget = transmissionTarget;
-            postCommandEvenIfDisabled(new Runnable() {
-
-                @Override
-                public void run() {
-                    finalTransmissionTarget.initInBackground(mContext, mChannel);
-                }
-            });
             return transmissionTarget;
         }
     }
 
+    /**
+     * Unconditionally create a new transmission target at root level.
+     *
+     * @param transmissionTargetToken the token.
+     * @return the created target.
+     */
+    private AnalyticsTransmissionTarget createAnalyticsTransmissionTarget(String transmissionTargetToken) {
```

Comment:
Since you split it off, can you be more explicit in the method name? `createAnalyticsTransmissionTargetAtRoot` is my suggestion.